CONTRIBUTING.md

```diff
@@ -48,6 +48,8 @@ If you get stuck we have two publicly-archived mailing lists: the https-everywhe
 
 You can also find more information on about HTTPS Everywhere on our [FAQ](https://www.eff.org/https-everywhere/faq) page.
 
+Also, please remember that this Project is Governed by [EFF's Public Projects Code of Conduct](https://www.eff.org/pages/eppcode).
```

Comment:
Is capital `Governed` here on purpose (e.g. it has a special meaning legally)?

Reply:
Nope, fixed!